src/fairchem/core/modules/scaling/fit.py

```diff
@@ -205,7 +206,7 @@ def index_fn(name: str = name) -> None:
 
         torch.save(
             {
-                x[0].replace(".scale_factor", ""): x[1]
+                re.sub("^backbone.", "", x[0].replace(".scale_factor", "")): x[1]
```

Comment:
Won't this break scaling factors for non-hydra models?

Reply:
It shouldnt, it only adds a rule ontop of whats there, to replace "backbone." from start of line 

There is tests in this PR for both gemnet regular and gement hydra

Reply:
I get it now :)